<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <property name="spacing">
     <number>1</number>
    </property>
    <item row="0" column="0" rowspan="2">
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(50, 50, 50);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="toolButtonAdd">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="text">
          <string>合成</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/add_on.png</normaloff>
           <disabledoff>:/add_on.png</disabledoff>:/add_on.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonMint">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="text">
          <string>铸造</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/mint_off.png</normaloff>:/mint_off.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonOffer">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="text">
          <string>offer</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/offer_off.png</normaloff>:/offer_off.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>244</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonStart">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="text">
          <string>开始-A</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/start_on.png</normaloff>:/start_on.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frameSet">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QFrame" name="frame_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>45</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>提示：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditMessage">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>已启动</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>108</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="save"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="documentHelp"/>
    <addaction name="separator"/>
    <addaction name="composeHelp"/>
    <addaction name="mintHelp"/>
    <addaction name="offerHelp"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="software"/>
   </widget>
   <widget class="QMenu" name="menulanguage">
    <property name="title">
     <string>language</string>
    </property>
    <addaction name="chinese"/>
    <addaction name="actionEN"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>注册</string>
    </property>
    <addaction name="logon"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menulanguage"/>
  </widget>
  <action name="save">
   <property name="text">
    <string>保存配置</string>
   </property>
  </action>
  <action name="composeHelp">
   <property name="text">
    <string>合成视频教程</string>
   </property>
   <property name="toolTip">
    <string>合成视频教程</string>
   </property>
  </action>
  <action name="mintHelp">
   <property name="text">
    <string>铸造视频教程</string>
   </property>
   <property name="toolTip">
    <string>铸造视频教程</string>
   </property>
  </action>
  <action name="offerHelp">
   <property name="text">
    <string>offer视频教程</string>
   </property>
   <property name="toolTip">
    <string>offer视频教程</string>
   </property>
  </action>
  <action name="documentHelp">
   <property name="text">
    <string>文档教程</string>
   </property>
   <property name="toolTip">
    <string>文档教程</string>
   </property>
  </action>
  <action name="logon">
   <property name="text">
    <string>注册信息</string>
   </property>
   <property name="toolTip">
    <string>注册信息</string>
   </property>
  </action>
  <action name="software">
   <property name="text">
    <string>关于本软件</string>
   </property>
   <property name="toolTip">
    <string>关于本软件</string>
   </property>
  </action>
  <action name="chinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>中文</string>
   </property>
   <property name="toolTip">
    <string>中文</string>
   </property>
  </action>
  <action name="actionEN">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>EN</string>
   </property>
   <property name="toolTip">
    <string>EN</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
